client/components/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+
```

Comment:
why not to add all old code and comment it out?

Reply:
Added files for what I need, but can add all the code too

Reply:
yes please, so we know what we'll need to do